privacyidea/lib/resolvers/LDAPIdResolver.py

```diff
@@ -983,14 +998,23 @@ def testconnection(cls, param):
         size_limit = int(param.get("SIZELIMIT", 500))
         serverpool_rounds = int(param.get("SERVERPOOL_ROUNDS") or SERVERPOOL_ROUNDS)
         serverpool_skip = int(param.get("SERVERPOOL_SKIP") or SERVERPOOL_SKIP)
-        if is_true(param.get("TLS_VERIFY")) \
-                and (ldap_uri.lower().startswith("ldaps") or
-                                    param.get("START_TLS")):
-            tls_version = int(param.get("TLS_VERSION") or ssl.PROTOCOL_TLSv1)
-            tls_ca_file = param.get("TLS_CA_FILE") or DEFAULT_CA_FILE
-            tls_context = Tls(validate=ssl.CERT_REQUIRED,
-                              version=tls_version,
-                              ca_certs_file=tls_ca_file)
+        if ldap_uri.lower().startswith("ldaps") or param.get("START_TLS"):
+            tls_force_version = is_true(param.get("TLS_FORCE_VERSION")) or False
```

Comment:
It is not clear to me, why we need the additional parameter TLS_FORCE_VERSION.
I was under the impression, that setting a distinct version actually forces to this version.
Lets talk